src/integrations/authentication/module.ts

```diff
@@ -1,6 +1,7 @@
 
-export * from './definitions/errors.js';
 export * from './definitions/interfaces.js';
 export * from './definitions/types.js';
+export * from './errors/AuthenticationError.js';
```

Comment:
```suggestion
export { default as AuthenticationError } from './errors/AuthenticationError.js';
```

This also goes for all the other errors in this and other modules.